docs/DeveloperGuide.md

```diff
@@ -165,7 +165,7 @@ The first section is facilitated by the following classes:
 The following is a simple class diagram of the three classes:
 <p align="center">
 
-![](diagrams/ParseAddRelatedClassesDiagram.jpg)
+![](diagrams/ParseAddRelatedClassesDiagram.png)
```

Comment:
The class diagram seems to be different from what was taught in class. Perhaps there can be different setting to it? 

If there isn't can we consider not using PlantUML for class diagram?